yarn.lock

```diff
@@ -729,6 +729,13 @@
   dependencies:
     regenerator-runtime "^0.13.2"
 
+"@babel/runtime@^7.10.2":
```

Comment:
Lockfile is up to date? I'm surprised to see this change

Reply:
It's a dependency of `gatsby-plugin-google-analytics`

Reply:
Dang that's annoying. This is why I don't like plugins haha